src/Components/Form/Form.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react';
+import ApiRequest from '../ApiRequest/ApiRequest';
```

Comment:
this will come as a const from common/variables

Reply:
mhm I do not understand that

Reply:
I removed that component, basicly I dont need it